sources/packages/api/src/services/auth/token-cache.service.models.ts

```diff
@@ -2,3 +2,11 @@ export interface TokenCacheResponse {
   accessToken: string;
   expiresIn: number;
 }
+export interface ATBCAuthTokenResponse {
+  accessToken: string;
+}
+
+export interface ATBCHeader {
+  headers?: { Authorization: string };

```

Comment:
I believe that headers should not be optional, right? Please consider removing the "?".

Reply:
I am resuing the interface for login and other ATBC endpoints, okay, I will create separate header interface